karma/makeconf.js

```diff
@@ -29,7 +30,7 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
         throw new Error('Unknown browser');
       }
     } else if (process.platform === 'darwin') {
-      browsers = ['ChromeWebRTC', 'FirefoxWebRTC', 'Safari'];
+      browsers = ['ChromeWebRTC', 'FirefoxWebRTC', 'Safari', 'Electron'];
```

Comment:
@makarandp0 Nitpick: Add `Electron` before `FirefoxWebRTC` to maintain alphabetical order?